app/controller/toolbar/MinimizedWindows.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * This class is the controller for the MinimizedWindows.
+ */
+Ext.define('CpsiMapview.controller.toolbar.MinimizedWindows', {
+    extend: 'Ext.app.ViewController',
+
+    alias: 'controller.cmv_minimized_windows',
```

Comment:
A hint that this is related to a toolbar view would be nice to have,s o it is more intuitive.
```suggestion
    alias: 'controller.cmv_minimized_windows_toolbar',
```

Reply:
solved in https://github.com/compassinformatics/cpsi-mapview/pull/179/commits/0b106a127d97601d29ea31d149448e09874ec88e